src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
+  }
+}
+
+.PermissionsCard-subhead {
+  color: #737373;
```

Comment:
We try to always use standard Photon colors and if the mocks stray from that then you should ask Phil if it was intentional or not. The easiest way to find the color var is to [ripgrep](https://github.com/BurntSushi/ripgrep) the hex (in lower case). This is how I found it:

```
$ rg 737373 node_modules/
node_modules/photon-colors/colors.scss
56:$grey-50: #737373;
...
```

You can then add the var like:

```css
@import "~photon-colors/colors";

.PermissionsCard-subhead {
  color: $grey-50;
}
```

Reply:
Thanks for the ripgrep tip - useful!